inventory-generation/notifications/main.yaml

```diff
@@ -15,11 +15,6 @@
       include_vars:
         file: "{{ directory }}/engagement.json"
 
-    - name: "Check for existing email templates"
```

Comment:
Why is this removed? It appears that whenever this playbook runs, it will now overwrite any local changes within the IaC repos, and that's probably not what we'd want(?) - unless this check was moved elsewhere and I just missed it?

Reply:
Yes, it should be here and will add it back in.

During testing, at one point I was seeing `group_vars` all being created without the `*.j2` files being processed. The reason was I was pointed to a branch that simply didn't have the jinja2 templates yet.

As long as we get he email-templates jinja2 files merged in before dispatcher picks up these new jobs we will be good to go!

I believe this is an area to build on in the very near future, such as adding a `reset_email_templates` variable here to allow automation to set templates back to default without having to manually delete the `group_vars/all` folder.